src/utils/emailController.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Handles API requests to the proxy server for contact form
+ */
+
+import axios from 'axios';
+
+const apiBase =
+  process.env.GRIDSOME_LANDING_API_HOST ||
+  (process.env.NODE_ENV === 'production'
+    ? 'https://landingapi.hackthe6ix.com'
+    : 'http://localhost:6970');
```

Comment:
Let's change this to use an appropriate env (you can remove the stuff after `||`.
https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser

Reply:
@jodiex use `GATSBY_APP_API_URL`